homeworks/ekaterina.sharova_rogerosha/4-basics/basics.js

```diff
@@ -0,0 +1,41 @@
+// sum = num1 + num2;
+// difference = num1 - num2;
+// multiple = num1 * num2;
+// division = num1 / num2;
+// residue = num1 % num2;
+// exponent = num1 ** num2;
+
+function calculation() {
+    const num1 = parseFloat(document.getElementById('num1').value);
+    const num2 = parseFloat(document.getElementById('num2').value);
+    const operator = document.getElementById('operator').value;
+    let result;
+
+    switch (operator) {
+        case '+':
+            result = num1 + num2;
+            break;
+        case '-':
+            result = num1 - num2;
+            break;
+        case '*':
+            result = num1 * num2;
+            break;
+        case '/':
+            result = num1 / num2;
+            break;
+        case '%':
+            result = num1 % num2;
+            break;
+        case '**':
+            result = num1 ** num2;
+            break;
+        default:
+            result = 'Wrong operator. Please, try again.';
+    }
+    document.getElementById('result').textContent = result;
+}
+
+module.exports = {
+    calculation,
+};
```

Comment:
You can use ` calculation()` at the end to start your function instead of `module.exports`